zenoh-plugin-ros2dds/src/node_info.rs

```diff
@@ -29,14 +29,24 @@ pub struct MsgPub {
     pub name: String,
     #[serde(rename = "type")]
     pub typ: String,
+    // List of DDS Writers declared by 1 Node for a same topic
+    // Issue #27: usually only 1 Writer, but may happen that 1 Node declares several Publishers
+    //            on the same topic. In this case `ros2 node info <node_id>` still shows only 1 Publisher,
+    //            but all the writers can be seen with `ros2 topic info <topic_name> -v`.
+    //            Hence the choice here to aggregate all the Writers in 1 Publisher representation.
+    //            The Publisher is declared undiscovered only when all its Writers are undiscovered.
```

Comment:
General question,
What are the differences between Writers and Publishers In this text ?
Are Publishers a Zenoh concept, and Writers a ROS2 concept ?

Reply:
Answering on behalf of @JEnoch . Here *Writers* are related to DDS while *Publisher* are related to Zenoh.

Reply:
To be more precise, in the context of this code:
- *Writer* means *DDS DataWriter*
- *Publisher* means *ROS 2 Publisher*, which happens to map in this plugin to a Zenoh Publisher.